wsf_scraping/spiders/base_spider.py

```diff
@@ -16,15 +16,16 @@ def on_error(self, failure):
 
         if failure.check(HttpError):
             response = failure.value.response
-            self.logger.error('HttpError on %s', response.url)
+            self.logger.warning('HttpError (%s) on %s',
+                                response.url, response.status)
 
         elif failure.check(DNSLookupError):
             request = failure.request
-            self.logger.error('DNSLookupError on %s', request.url)
+            self.logger.warning('DNSLookupError on %s', request.url)
```

Comment:
Even though I understand why you are doing that, I am still wondering whether timeout or an error such as dnslookuperror should still be considered normal. 

Or in other words, if we fail to retrieve all requests, so all throw a time out error because of a change we made, we will not receive a notification which is not optimal. 

Let me know what you think.

Reply:
Well, it's a complicated issue. We will probably always have a few requests that won't find there way. That beeing said, it is no reason to consider these errors normal. By assigning them to the warning category rather than the error category, we get the benefit of logging them, enabling the possibility of using them as a more relevant thing: metrics. 

By considering them as error, we don't have relevant information: we know that a particular url failed.

By monitoring the warnings, we clear the error output, leaving us with real errors, and we have stats abotu how many errors of each type we actually encounter for every organisation we scrape, which in my opinion is more important.

Reply:
I will leave this to you and hunter since you know better 👍